openadapt/models.py

```diff
@@ -61,6 +61,9 @@ def processed_action_events(self):
 
 class ActionEvent(db.Base):
     __tablename__ = "action_event"
+    _text_sep = config.TEXT_SEP
+    _text_name_prefix = config.TEXT_NAME_PREFIX
+    _text_name_suffix = config.TEXT_NAME_SUFFIX
```

Comment:
Please remove these and refer to `config.TEXT_*` directly where these are used

Reply:
Done!